swift-ci/master/amazon-linux/2/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM amazonlinux:2
+FROM swift:5.7-amazonlinux2
```

Comment:
should this be pinned to 5.7 or use the latest release?

Reply:
Pinned makes sense to me given that we support releases up to 12 months back. So 5.7/5.8/.5.9. Though ideally we'd have a build for 5.8 + 5.9 going as well.